views/metrics/components/chart-wrapper/index.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box } from '@interest-protocol/ui-kit';
+import { FC, PropsWithChildren } from 'react';
+
+import { ChartHeaderProps } from './chart.types';
+import ChartHeader from './header';
+
+const ChartWrapper: FC<PropsWithChildren<ChartHeaderProps>> = ({
+  children,
+  ...props
+}) => (
+  <Box
+    width="100%"
+    display="flex"
+    borderRadius="xs"
+    overflowY="hidden"
+    color="onSurface"
+    gridColumn="1/-1"
```

Comment:
Remove it or change to display grid, cuz there is no effect for display flex

Reply:
Bad review